rosdep/python.yaml

```diff
@@ -892,6 +894,11 @@ python-libpgm-pip:
   ubuntu:
     pip:
       packages: [libpgm]
+python2-lttngust:
```

Comment:
Unfortunately, babeltrace is not on pypi. I surmise that's because it's built as part of a 'C' project.

It gets worse in that python3-babeltrace is not usually available on Ubuntu either, because it's on a PPA.

I'll just remove it.